lib/services/devices/deviceService.js

```diff
@@ -173,6 +173,9 @@ function mergeDeviceWithConfiguration(fields, defaults, deviceData, configuratio
     if (configuration && configuration.entityNameExp !== undefined) {
```

Comment:
Changes already included in existing CNR entry?

Reply:
fixed in https://github.com/telefonicaid/iotagent-node-lib/pull/1508/commits/93c6952ed7e740ec4bcbf544e05de4c018342a5b